pep-0394.txt

```diff
@@ -5,104 +5,181 @@ Last-Modified: $Date$
 Author: Kerrick Staley <mail@kerrickstaley.com>,
         Nick Coghlan <ncoghlan@gmail.com>,
         Barry Warsaw <barry@python.org>,
-        Petr Viktorin <encukou@gmail.com>
+        Petr Viktorin <encukou@gmail.com>,
+        Miro Hrončok <miro@hroncok.cz>,
 Status: Active
 Type: Informational
 Content-Type: text/x-rst
 Created: 02-Mar-2011
-Post-History: 04-Mar-2011, 20-Jul-2011, 16-Feb-2012, 30-Sep-2014, 28-Apr-2018
+Post-History: 04-Mar-2011, 20-Jul-2011, 16-Feb-2012, 30-Sep-2014, 28-Apr-2018,
+              12-Apr-2019
 Resolution: https://mail.python.org/pipermail/python-dev/2012-February/116594.html
 
 
 Abstract
 ========
 
 This PEP provides a convention to ensure that Python scripts can continue to
-be portable across ``*nix`` systems, regardless of the default version of the
-Python interpreter (i.e. the version invoked by the ``python`` command).
+be portable across ``*nix`` systems, regardless of the version of the Python
```

Comment:
Perhaps:

This PEP outlines the behavior of Python scripts when the ``python`` command is invoked. Depending on a distribution or system configuration, ``python`` may or may not be installed. If ``python`` is installed its target interpreter may refer to ``python2`` or ``python3``. End users may be unaware of this inconsistency across Unix-like systems. This PEP's goal is to reduce user confusion about what ``python`` references and what will be the script's behavior.

The recommendations in the next section of this PEP will outline the behavior when:
- using virtual environments
- cross-platform scripts with shebangs for either ``python2`` or ``python3`` 

The PEP's goal is to clarify the behavior for script end users, distribution providers, and script maintainers / authors.